fuzzbench/run_experiment.py

```diff
@@ -42,8 +45,13 @@ def run_experiment():
 def main():
     """Set up Redis connection and start the experiment."""
     redis_connection = redis.Redis(host="queue-server")
+    config_path = environment.get('EXPERIMENT_CONFIG', 'config.yaml')
```

Comment:
Shouldn't this use experiment_utils to get config file. Why use env var ? for tests, we just mock var.

Reply:
> Shouldn't this use experiment_utils to get config file.

Do you suggest that we put the config getting logic under `common/experiment_utils.py` rather than `common/config_utils.py`?

> Why use env var ? for tests, we just mock var.

For end-to-end test, we cannot monitor the experiment and can only check the outcome when it finishes. However, to let our end-to-end test run with a specified config, it should be set in the compose/ folder. Thus, we choose to use a global environment variable to do this.